Задание 2:
Программа выдат ошибку по пичине не объявленного массива intArray[8]
и переменная d равна нуля, а на ноль делить нельзя.

Задание 3:
Лишнее throws Exception при объявлении метода (public static void main(String[] args) throws Exception).

Лишняя обработка: catch (NullPointerException ex) {
System.out.println("Указатель не может указывать на null!");.

Лишнее throws FileNotFoundException при объявлении метода public static void printSum(Integer a, Integer b) throws FileNotFoundException {
System.out.println(a + b);.




